db/migrate/20220705164740_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the ERD, the foreign key in the posts table references the `author` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :posts, :author, foreign_key: { to_table: :users }, null: false
  end
```